src/components/EmptyTableInfo.js

```diff
@@ -0,0 +1,23 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import styles from './EmptyTableInfo.module.css'
+
+const EmptyTableInfo = ({ action, header, icon, text }) => (
+    <>
+        <div className={styles.wrapper} data-test="data-test-empty-table-info">
+            <div className={styles.icon}>{icon}</div>
+            <p className={styles.header}>{header}</p>
+            <p className={styles.text}>{text}</p>
+            <div className={styles.action}>{action}</div>
+        </div>
+    </>
+)
+
+EmptyTableInfo.propTypes = {
+    action: PropTypes.node,
+    header: PropTypes.node,
+    icon: PropTypes.node,
+    text: PropTypes.node,
+}
+
+export default EmptyTableInfo
```

Comment:
This component will perhaps be included in `UI`, but I haven't landed on a standardized usage yet (not sure there is one). So for now, this component lives here with the possibility of moving to `UI` in the future.